addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -47,29 +47,23 @@ options.Class.include({
 });
 
 options.registry.background.include({
+    background: async function (previewMode, widgetValue, params) {
+        if (previewMode === 'reset' && this.videoSrc) {
+            return this._setBgVideo(false, this.videoSrc);
+        }
+        // Caching _super because we need it asynchronously.
```

Comment:
I'd remove the comment, we do that all the time, I think if we explain it everytime it will be painful... ;)